### ISSUE TITLE
Part of the 3d body disappers when creating a sketch on one of its faces

### ISSUE THREAD

**Issue by delhub** [Ondsel-Development/FreeCAD#112]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

While walking through a tutorial from [FreeCAD Beginner Tutorial #1
](https://www.youtube.com/watch?v=VZ16lty5ADg) I ran across an issue where part of the 3D body disappears upon creating a sketch on one face all body parts "above" that face disappear.  

### Full version info

```shell
OS: Windows 11 build 22631
Word size of Ondsel: 64-bit
Version: 2024.2.0dev.37914 (Git)
Build type: Release
Branch: flavor
Hash: 88e76e1d516654ee4a020cf4ad676853ca4d2f31
Python 3.11.9, Qt 5.15.13, Coin 4.0.2, Vtk 9.2.6, OCC 7.7.2
Locale: English/United States (en_US)
Installed mods: 
  * Ondsel-Lens 2024.5.6.01
```

### Subproject(s) affected?

None

### Anything else?

Attached is a file of the 3D object with the face selected to be sketched on.
![3D object](https://github.com/Ondsel-Development/FreeCAD/assets/16364999/611449c9-3ede-4c45-9b88-9b071354a441)

When creating a sketch on this face, one is presented with the following:
![Screenshot 2024-07-05 220248](https://github.com/Ondsel-Development/FreeCAD/assets/16364999/2372aff8-603f-4511-8022-855921bd37ba)

Note that the presentation of the XY-plane is rotated 180 degrees (this is strange to me, but may not be germaine to this issue).  However, when pressing the '0' key this view is displayed showing the parts that are missing from the object: 
![Screenshot 2024-07-05 221353](https://github.com/Ondsel-Development/FreeCAD/assets/16364999/71dfdf1f-cfc6-4524-bed1-3e9a572a9d9d)

Note that this is my 4th attempt at trying to use FreeCAD - still frustrating - (but I'm willing to give it one more shot), hence my attempt to use Ondsel (you folks are on the right track).

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by delhub** [comment]
This issue is partially resolved by unchecking all "Visibility Automation" settings under "Sketcher/Display" preferences - all except for "Show objects used for extend geometry" and "Show objects that the sketch is attached to".  However, clicking the "Apply to existing sketches" made no difference if the sketch already existed.  One had to set the setting, then delete the existing sketch and restart the sketch for the settings to take effect.

**Comment by BlenderManiac40** [comment]
Try View Section button, shortcut q s on keyboard to change to whole object view. 
![Screenshot (4)](https://github.com/Ondsel-Development/FreeCAD/assets/68447017/0bb38865-2c26-4f56-a433-b584c7ecde80)

**Comment by delhub** [comment]
Thank you.  I should have found that myself.   I truly don't remember, now, if I had tried that.  I would have thought that after clicking the "Apply to existing sketches" in the preferences it would have fixed the proble.  I do know that I used that today on another project to reveal a sketch that was hidden behind other objects.  So, it works for that.  I'm satisfied with the available options at the moment, so we can close this issue.  Thanks, again.
